src/state.tsx

```diff
@@ -50,7 +50,8 @@ const generateInitialEquipment = () => {
   return initialEquipment;
 };
 
-export const generateEmptyPlayer: () => Player = () => ({
+export const generateEmptyPlayer: (name?: string) => Player = (name?: string) => ({
```

Comment:
You can list the params only once with this and then not worry about writing it twice if they change
```ts
export const generateEmptyPlayer = (name?: string): Player => ({
    ...
});
```